swift-sdk/IterableFlexMessagingManagerProtocol.swift

```diff
@@ -5,5 +5,5 @@
 import Foundation
 
 @objc public protocol IterableFlexMessagingManagerProtocol {
-    
+    func start()
 }
```

Comment:
Is this a placeholder to test before implementing the other functionality?

Reply:
Sort of, it is the convention with how we design managers in the iOS SDK. See `InAppManager` for the reference.